packages/core/src/mergeClasses.test.ts

```diff
@@ -9,6 +9,8 @@ const options: MakeStylesOptions = {
   renderer: createDOMRenderer(document),
 };
 
+const EMPTY_STRING_HASH = '0000000';
```

Comment:
Not sure that these changes in tests are good. I propose:

- run them in production mode
  ```ts
  beforeEach(() => {
    process.env.NODE_ENV = 'test';
  });
  ```
- have a separate test for dev